.gitignore

```diff
@@ -196,4 +196,4 @@ hs_err_pid*
 coverage
 
 #file for start localhost#
-tomcat.sh
\ No newline at end of file
+tomcat.sh
```

Comment:
seem like `tomcat.sh` should be excluded but it is still on the changelist. should it be here or not?